.golangci.yml

```diff
@@ -280,6 +296,8 @@ issues:
     # The errcheck linter already checks for unhandled errors so we can disable the equivalent
     # lint by gosec.
     - "G104: Errors unhandled"
+    # Random numbers here are not used in a security-sensitive context.
```

Comment:
Good call--I did some weird workarounds to use crypto/rand in a previous PR and it was all pretty silly.